src/rlookup.h

```diff
@@ -135,8 +135,11 @@ typedef struct {
 #define RLOOKUP_F_OEXCL 0x01   // Error if name exists already
 #define RLOOKUP_F_OCREAT 0x02  // Create key if it does not exit
 
-/** Force this key to be the output key, bypassing the sort vector */
-#define RLOOKUP_F_OUTPUT 0x04
```

Comment:
Didn't find where we use this flag, or any commit in the entire RediSearch lifetime that includes it.